stackslib/src/main.rs

```diff
@@ -1017,6 +1021,224 @@ simulating a miner.
         process::exit(0);
     }
 
+    if argv[1] == "analyze-sortition-mev" {
+        if argv.len() < 7 || (argv.len() >= 7 && argv.len() % 2 != 1) {
+            eprintln!(
+                "Usage: {} /path/to/burnchain/db /path/to/sortition/db /path/to/chainstate/db start_height end_height [advantage_miner advantage_burn ..]",
+                &argv[0]
+            );
+            process::exit(1);
+        }
+
```

Comment:
Can you move the implementation of `analyze-sortition-mev` to a new function in this file, and then just invoke that here? The giant `if` switch statement is pretty unwieldy and we should stop adding big code blocks to it.

Reply:
Sure